pyscriptjs/src/main.ts

```diff
@@ -285,24 +306,32 @@ export class PyScriptApp {
 
         // inject `define_custom_element` and showWarning it into the PyScript
         // module scope
-        // eventually replace the setHandler calls with interpreter._remote.setHandler i.e. the ones mentioned below
-        // await interpreter._remote.setHandler('define_custom_element', Synclink.proxy(define_custom_element));
-        // await interpreter._remote.setHandler('showWarning', Synclink.proxy(showWarning));
-        interpreter._unwrapped_remote.setHandler('define_custom_element', define_custom_element);
-        interpreter._unwrapped_remote.setHandler('showWarning', showWarning);
+        if (interpreter.useWorker) {
+            // IMPLEMENT ME!
+            // await interpreter._remote.setHandler('define_custom_element', Synclink.proxy(define_custom_element));
+            // await interpreter._remote.setHandler('showWarning', Synclink.proxy(showWarning));
+        } else {
+            interpreter._unwrapped_remote.setHandler('define_custom_element', define_custom_element);
+            interpreter._unwrapped_remote.setHandler('showWarning', showWarning);
+        }
+
         const pyscript_module = (await interpreter.pyimport('pyscript')) as Synclink.Remote<
             PyProxy & { _set_version_info(string): void }
         >;
         await pyscript_module._set_version_info(version);
         await pyscript_module.destroy();
 
         // import some carefully selected names into the global namespace
-        await interpreter.run(`
-        import js
-        import pyscript
-        from pyscript import Element, display, HTML
-        pyscript._install_deprecated_globals_2022_12_1(globals())
-        `);
+        if (this.interpreter.useWorker) {
+            // FIXME
+        } else {
```

Comment:
In #1338 I move this into `loadInterpreter` which should resolve this issue. But isn't the code the same in both branches anyways?

Reply:
yes, fixed now